ckanext/dcat_usmetadata/templates/organization/read.html

```diff
@@ -0,0 +1,43 @@
+{% extends "organization/read_base.html" %}
```

Comment:
Have no idea about this new file, so just skipping to review this.

Reply:
This makes no impact on Inventory app but it suppose to change "Add dataset" button link on organization page. I've opened an issue about that here https://app.zenhub.com/workspaces/datagov-multi-tenant-ckan-5cf12b601437ff1e943f0aa2/issues/gsa/datagov-ckan-multi/476